packages/app/src/runner/useRunnerStyle.ts

```diff
@@ -16,19 +16,13 @@ const reporterWidth = ref<number>(0)
 const specListWidth = ref<number>(0)
 
 interface UseRunnerUI {
-  initialSpecsListWidth: number
-  initialReporterWidth: number
+  initialSpecsListWidth: number | null
+  initialReporterWidth: number | null
 }
 
-export const useRunnerStyle = ({
-  initialReporterWidth,
-  initialSpecsListWidth,
-}: UseRunnerUI = {
-  initialReporterWidth: runnerConstants.defaultReporterWidth,
-  initialSpecsListWidth: runnerConstants.defaultSpecListWidth,
-}) => {
-  reporterWidth.value = initialReporterWidth
-  specListWidth.value = initialSpecsListWidth
+export const useRunnerStyle = (initialValues?: UseRunnerUI) => {
+  reporterWidth.value = initialValues?.initialReporterWidth ?? runnerConstants.defaultReporterWidth
```

Comment:
We will still support node 12 so we prob shouldn't be doing optional chaining since this was introduced in node 14.